### PR TITLE
fix setting exist email #2

### PR DESCRIPTION
**Pull request by votuanthanh**

### CODE REVIEW

**Comment by trinhductoan** [comment]
OK

File: resources/assets/js/poll.js [hunk 1]
```diff
@@ -961,7 +961,8 @@ $(document).ready(function () {
         $.each(pollData.config.setting, function (index, value) {
             $("[name='setting\\[" + value + "\\]']").bootstrapSwitch({
                 'onText' : pollData.message.on,
-                'offText' : pollData.message.off
+                'offText' : pollData.message.off,
```
Comment (tuanpht) [inline resources/assets/js/poll.js]: sai 1 tí convention

File: resources/views/layouts/poll_setting.blade.php [hunk 2]
```diff
@@ -51,31 +52,69 @@
...
+                    <div class="st">
+                        <label class="radio-inline radio-setting-required">
+                            {{ Form::radio('setting_child[required]', config('settings.setting.required_name'), true) }}
+                            {{ trans('polls.label.setting.required_name') }}
+                        </label>
+                    </div>
+
+                    <div class="st">
+                        <label class="radio-inline radio-setting-required">
+                                {{
```
Comment (tuanpht) [inline resources/views/layouts/poll_setting.blade.php]: indent

**Comment by at-diephq** [comment]
OK

**Pull request merged**